src/config/IAppConfig.ts

```diff
@@ -137,7 +137,7 @@ export interface IServerConfig {
     query_sets_of_genes: string | null;
     skin_quick_select_buttons: string | null;
     base_url: string | null;
-    user_email_address: string;
+    user_display_name: string;
```

Comment:
does this have an associated backend change?

Reply:
@alisman Yes. It's this change: https://github.com/cBioPortal/cbioportal/pull/9986/files#diff-d1ee85cbb7548f7a3c90d6d7468d5ce8b1f7a4960d760968d533360ceeadd6e1R174

Reply:
ok, will wait till it's merged and deployed next week to merge this